corehq/apps/cloudcare/static/cloudcare/js/formplayer/utils/utils.js

```diff
@@ -501,7 +501,7 @@ hqDefine("cloudcare/js/formplayer/utils/utils", function () {
                         stopSyncInterval();
                     }
                     if (shouldSync() && FormplayerFrontend.permitIntervalSync) {
-                        FormplayerFrontend.getChannel().request("interval_sync-db", urlObject.appId);
+                        FormplayerFrontend.trigger("interval_sync-db", urlObject.appId);
```

Comment:
A key difference that i'm aware of is that `request` can return values while `trigger` (`events`) don't. 

Marionette documents principles of when to use [events](https://marionettejs.com/docs/master/backbone.radio.html#when-to-use-events) and when to use [requests](https://marionettejs.com/docs/master/backbone.radio.html#when-to-use-requests). I mainly decided to use trigger to follow what exists and because it doesn't return anything.